### ISSUE TITLE
Add parseBuffer method

### ISSUE THREAD

**Issue by jbzcooper** [gatewayapps/node-tnef#6]
I would like to parse a buffer of data rather than reading from the file system.

**Comment by johnmurphy01** [comment]
Hey @jbzcooper that sounds like it could be a useful method, I'll see if I can add it. Shouldn't be very difficult

**Comment by johnmurphy01** [comment]
@jbzcooper  What are you thinking the output would be? Do you want it to still save to a file path or do you want the data back as a `Buffer`?

**Comment by jbzcooper** [comment]
@johnmurphy01 for my purposes a parsed object with attachments as buffers would be ideal. Basically what your 'Decode' method (line 165) returns would be great.

**Comment by johnmurphy01** [comment]
@jbzcooper one more question: I'm assuming you're using this within a NodeJS project. If that's the case, then once I push my changes and deploy you should be able to:

```
var tnef = require('node-tnef')

tnef.parseBuffer([your buffer of data], function (err, content) {
    // content would contain the result data as a Buffer
   // from here you can write to a file, evaluate the contents(ex. content.Attachments or content.Body)
})
```

If you need anything additional, please let me know

**Comment by jbzcooper** [comment]
Woah! Sweet man! Thank you!!!

**Comment by johnmurphy01** [comment]
I've made my changes but still need to test this before we can close

**Comment by jbzcooper** [comment]
Oh sorry! 😄

**Comment by johnmurphy01** [comment]
@jbzcooper I've just published version `1.3.0` of the package. Let me know if you have any issues using the new method
